integration_tests/ipp/container/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM zgrab2_service_base:latest
+
+RUN apt-get update
+RUN apt-get install -y cups
+RUN apt-get install -y cups-pdf
+RUN echo "ServerAlias *" >> /etc/cups/cupsd.conf
+RUN echo "FileDevice yes" >> /etc/cups/cupsd.conf
+RUN sed -i 's/Order allow,deny/Order allow,deny\'$'\n  Allow all/g' /etc/cups/cupsd.conf
+RUN sed -i 's/Listen localhost:631/Listen 0.0.0.0:631/g' /etc/cups/cupsd.conf
+RUN service cups restart
+
+# Try to make it act more container-y -- remove it from init.d and just run the daemon as the entrypoint
+#RUN service cupsd stop
+#RUN update-rc.d -f cupsd remove
```

Comment:
Same as above; the intent here is to not only stop the service, but prevent it from being started automatically.